tests/constants.ts

```diff
@@ -44,9 +44,9 @@ console.debug(`UXD PROGRAM ID == ${uxdProgramId}`);
 export const uxdClient = new UXDClient(uxdProgramId);
 
 // Used in mercurial vault tests
-export const SOLEND_USDC_DEVNET = new PublicKey(
+export const MERCURIAL_USDC_DEVNET = new PublicKey(
```

Comment:
maybe just mention in a comment that this is a solend USDC

Reply:
this is really a mercurial devnet USDC, they setup the vault for us becoz we can't get anymore solend usdc, just the naming is not updated until now :p